frontend/app/js/app.coffee

```diff
@@ -1,6 +1,9 @@
 angular.module('exercism', ['ui.bootstrap'])
 
 $ ->
+  if $('h1, h2, h3').length && document.title == 'exercism.io'
+    document.title = "#{$($('h1, h2, h3').get(0)).text()} - exercism.io"
```

Comment:
Ah, good catch. Added a commit to leave the homepage alone.  Initially I only looked at H1, but a lot of pages don't have H1. Some don't even have H2.  Checking all three seem to catch most of them.

I'm not a huge fan of doing it this way and would prefer to set realistic page titles on the server side so it could say "ruby test submission by xyz" etc, but thought this might be quicker for now.  If I have time I may add others which would supersede this.